fe/lib/components/LocationSuggest/LocationSuggest.tsx

```diff
@@ -70,6 +87,24 @@ export const LocationSuggest = forwardRef<HTMLInputElement, Props>(
       }
     };
 
+    const handleDetectLocationClicked = (input: GeoLocationInput | Error) => {
+      if (input instanceof Error) {
+        return (
+          <FieldMessage
```

Comment:
We should not return react component from this handle click function. There are a few options:
- We could either combine this error with the other graph errors
- Or specifically set-up state/ a new field message below for not being able to access the geolocation via navigator
- Or display the field error message within the `LocationSuggestInput component`

Reply:
>We should not return react component from this handle click function.

Any reason why not?